docs/your-first-snap.md

```diff
@@ -151,7 +151,7 @@ into your snap. If you look at `./stage`, you'll see a lot more files now:
     $ ls stage
     bin  etc  lib  usr  var
 
-### A copy Part
+### A Part using the dump plugin
```

Comment:
The capitalization here is a little off. How about just `A part using the dump plugin` ?